### PR TITLE
fix: issues about creating stickers in guild

### PR DESCRIPTION
**Pull request by ItsRqtl**
## About

This pull request is about (X), which does (Y).

## Checklist

- [x] The ``pre-commit`` code linter has been run over all edited files to ensure the code is linted.
- [ ] I've ensured the change(s) work on `3.8.6` and higher.

I've made this pull request: (check all that apply)
  - [ ] For the documentation
  - [ ] To add a new feature
  - [ ] As a general enhancement
  - [ ] As a refactor of the library/the library's code
  - [x] To fix an existing bug
  - [ ] To resolve #ISSUENUMBER

This is:
  - [ ] A breaking change

### CODE REVIEW

File: interactions/api/http/sticker.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ async def create_guild_sticker(
         :return: The new sticker data on success.
         """
         data = FormData()
-        data.add_field("file", file._fp, filename=file._filename)
+        data.add_field("file", file._fp, filename=file._filename, content_type="image/png")
```
Comment (sillyElvy) [inline interactions/api/http/sticker.py]: Does it work for all types? Also gifs?

**Comment by Damego** [comment]
Without content type its doesn't work?

File: interactions/api/http/sticker.py (hunk 1 above)
Comment (ItsRqtl) [reply interactions/api/http/sticker.py]: png and apng works, still figuring out lottie json.
gifs are not supported by discord (apng is the format of animated stickers)

**Comment by ItsRqtl** [comment]
set to draft until a solution for stickers with lottie json format.

**Review by sillyElvy — APPROVED** [review]
Seems good, thanks!

**Pull request merged**